index.html

```diff
@@ -0,0 +1,666 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://fonts.googleapis.com/css2?family=Maven+Pro:wght@600&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Heebo:wght@300&display=swap" rel="stylesheet">
+    <script src="https://kit.fontawesome.com/9a2f12287f.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="style.css">
+    <title>TNW</title>
+</head>
+
+<body>
+    <header>
+        <nav class="nav1">
+            <ul>
+                <li>News</li>
+                <li>Events <i class="fas fa-angle-down"></i> </li>
+                <li>Business <i class="fas fa-angle-down"></i> </li>
+                <li>AMAs</li>
+                <li>Spaces</li>
+                <li>Terms & Conditions</li>
+            </ul>
+            <ul>
+                <li>About</li>
+                <li>Advertise</li>
+                <li>Jobs</li>
+                <li>Contact</li>
+            </ul>
+        </nav>
+        <nav class="nav2">
+            <ul>
+                <li>LATEST</li>
+                <li>HARD FORK</li>
+                <li>PLUGGED</li>
+                <li>README</li>
+                <li>GROWTH QUARTERS</li>
+                <li>NEURAL</li>
+            </ul>
+            <div class="nav-search">
+                <i class="fas fa-search"></i>
+            </div>
+        </nav>
+        <nav class="mobile-nav">
+            <div class="mobile-nav2">
+                <img class="tnw-red" src="img/tnw-red.svg" alt="tnw logo">
+            </div>
+            <div class="mobile-nav1">
+                <div class="mobile-nav-search">
+                    <i class="fas fa-search gray"></i>
+                </div>
+                <div class="mobile-nav-more">
+                    <i class="fas fa-bars orange"></i>
+                </div>
+            </div>
+        </nav>
+    </header>
+    <main>
+        <section>
+            <div class="section1">
+                <img src="img/tnw.svg" alt="The Next Web Logo">
+                <div class="showcase-text">
+                    <h6 class="white showcase-header">INSIGHTS</h6>
+                    <h1 class="white showcase-title">Satoshi Nakamoto: ‘Bitcoin crosses $10K for first time in 2 months’
```

Comment:
- [ ] The TNW logo should disappear in the medium and small screen sizes.

| Yours | Original |
|:---:|:---:|
| ![image](https://user-images.githubusercontent.com/61479648/88699698-09907280-d0de-11ea-81ea-a02cdca9b192.png) | ![image](https://user-images.githubusercontent.com/61479648/88699728-144b0780-d0de-11ea-91eb-84ea1f896121.png) |

- [ ] On  the mobile, all the containers should have the same height

| Yours | Original |
|:---:|:---:|
|![image](https://user-images.githubusercontent.com/61479648/88699991-69871900-d0de-11ea-805d-fbd8cdcd4f5a.png) | ![image](https://user-images.githubusercontent.com/61479648/88700015-73a91780-d0de-11ea-97a9-5a723dce3d96.png) |